.htaccess

```diff
@@ -44,7 +45,6 @@ RewriteRule		^apple-touch-icon-precomposed.png$	-							[L,R=404]
 
 #top level sections
 RewriteRule		^history[/]*$					index.php?s=history				[NC,L]
-RewriteRule		^raceday[/]*$					index.php?s=raceday				[NC,L]
```

Comment:
Moved to WordPress

Reply:
So, you're relying on the auto-fallback for the permalinks to work, but... it looks like we might have 2 pages that are candidates for an exact match on "raceday" now.

Both: https://cmubuggy.org/news/wp-admin/post.php?post=7955&action=edit and your new page: https://cmubuggy.org/news/wp-admin/post.php?post=5910&action=edit claim to own that shorthand link.

I can't tell if this currently works (and it certainly does) because we are lucky or good (the "wrong" page is technically /news/reference/raceday and maybe the new "strict" bit is protecting us?).

Reply:
I actually don't think that either would be using that loose matching – shouldn't WordPress first match to pages that are specifically registered?

Separately, I did an experiment and tried to make another page with the slug `raceday`, but on publish it automatically changed to `raceday-2`. So that tells me that these pages are indeed distinctly identified (`raceday` and `reference/raceday`) because it wouldn't allow us to make pages with duplicate identifiers.